tools/distrib/python/make_grpcio_tools.py

```diff
@@ -71,7 +71,9 @@
 # will be added to include path when building grpcio_tools
 CC_INCLUDES = [
     os.path.join("third_party", "abseil-cpp"),
+    os.path.join("third_party", "protobuf"),
     os.path.join("third_party", "protobuf", "src"),
+    os.path.join("third_party", "protobuf", "upb"),
```

Comment:
Can we remove those two lines if we already have `os.path.join("third_party", "protobuf")`?

Reply:
If protobuf has other directories other than src and upb as an example, having a single protobuf is copying those unnecessary files?